libs/platform/core/src/services/page-head/default-page-head.service.ts

```diff
@@ -0,0 +1,73 @@
+import { ElementAttributes, ElementDefinition } from './page-head.model';
+import { PageHeadService } from './page-head.service';
+
+export class DefaultPageHeadService implements PageHeadService {
+  addElements(definitions: ElementDefinition | ElementDefinition[]): void {
+    if (!Array.isArray(definitions)) {
+      definitions = [definitions];
+    }
+
+    for (const definition of definitions) {
+      if (definition.name === 'html') {
+        this.updateHtmlElement(definition.attrs);
```

Comment:
I think we concluded, we don't want to handle 'html' element as a special case, but as a separate method? 🤔 
and we already have separate method I see `updateHtmlElement`